data_structures/src/superblock.rs

```diff
@@ -244,16 +335,14 @@ impl SuperBlockState {
     pub fn build_superblock(
         &mut self,
         block_headers: &[BlockHeader],
-        ars_pkh_keys_ordered: &[PublicKeyHash],
-        ars_ordered_bn256_keys: &[Bn256PublicKey],
+        ars_ordered_identities: ARSIdentities,
```

Comment:
Since `ars_ordered_identities` is now all the ARSIdentities I would call it `ars_identities`, does it make sense?